mailmerge/__main__.py

```diff
@@ -55,6 +57,11 @@
     type=click.IntRange(1, None),
     help="Start on message number INTEGER",
 )
+@click.option(
```

Comment:
There's a discussion in #92 about command line option vs server configuration parameter.  The conclusion of that discussion was to go with a server configuration.  Would you be willing to go that route?

Reply:
I’m not really a Python dev. Instrad I’m the Manager CSIRT of DIVD and needed a tool to warn of vulnerable Exchange servers. 
I’m maybe willing to look into it in the future, but not right now.

Reply:
I would be happy to help with this part.  It would be a new line a lot like this https://github.com/awdeorio/mailmerge/blob/develop/mailmerge/sendmail_client.py#L33

Reply:
I've managed to move the code to the sendmail object and read the configuration from the server configuration. This should be addressed now.